diff --git a/config.h b/config.h
index b41747f..4c2a304 100644
--- a/config.h
+++ b/config.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Inconsolata for Powerline:pixelsize=16:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -17,7 +17,8 @@ static int borderpx = 2;
  * 5: value of shell in config.h
  */
 static char shell[] = "/bin/sh";
-static char *utmp = NULL;
+static char tmux[] = "tmux";
+static char *utmp = tmux;
 static char stty_args[] = "stty raw pass8 nl -echo -iexten -cstopb 38400";
 
 /* identification sequence returned in DA and DECID */
@@ -32,7 +33,7 @@ static float chscale = 1.0;
  *
  * More advanced example: " `'\"()[]{}"
  */
-static char worddelimiters[] = " ";
+static char worddelimiters[] = " `'\"()[]{}:|-";
 
 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
@@ -80,7 +81,7 @@ static char termname[] = "st-256color";
  *
  *	stty tabs
  */
-static unsigned int tabspaces = 8;
+static unsigned int tabspaces = 4;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
@@ -109,6 +110,14 @@ static const char *colorname[] = {
 	/* more colors can be added after 255 to use with DefaultXX */
 	"#cccccc",
 	"#555555",
+
+    // cobalt
+    "#ffc600", // yellow
+    "#193549", // blue
+    "#35434d", // dusty blue
+    "#15232D", // dark blue
+    "#002f4f", // bg from sublime
+    "#002241", // bg from terminator
 };
 
 
@@ -117,7 +126,7 @@ static const char *colorname[] = {
  * foreground, background, cursor, reverse cursor
  */
 static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultbg = 263;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
diff --git a/st.info b/st.info
index b70fefa..14f1e46 100644
--- a/st.info
+++ b/st.info
@@ -46,7 +46,7 @@ st| simpleterm,
 	indn=\E[%p1%dS,
 	invis=\E[8m,
 	is2=\E[4l\E>\E[?1034l,
-	it#8,
+	it#4,
 	kel=\E[1;2F,
 	ked=\E[1;5F,
 	ka1=\E[1~,
